src/main/java/knightminer/simplytea/data/gen/RecipeGenerator.java

```diff
@@ -94,6 +96,18 @@ protected void buildRecipes(Consumer<FinishedRecipe> consumer) {
 		boil(consumer, teapot_water, teapot_hot);
 		boil(consumer, teapot_milk, teapot_frothed);
 
+		// empty teapots
+		ShapelessRecipeBuilder.shapeless(RecipeCategory.MISC, Items.WATER_BUCKET)
+											 .requires(teapot_water)
```

Comment:
I don't think this recipe actually works in practice; IIRC how I setup the teapot, this won't empty it, this will just drain 1/3 levels. It could probably be fixed but is probably more effort than its worth.